src/frontend/screens/MapScreen/track/SaveTrackScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {useState} from 'react';
+import {SafeAreaView, ScrollView, StyleSheet, View} from 'react-native';
+import {SaveTrackHeader} from './saveTrack/SaveTrackHeader';
+import {DiscardTrackModal} from './saveTrack/DiscardTrackModal';
+import {BottomSheet} from '../../../sharedComponents/BottomSheet/BottomSheet';
+import PhotoIcon from '../../../images/camera.svg';
+import DetailsIcon from '../../../images/details.svg';
+import TrackIcon from '../../../images/Track.svg';
+import {defineMessages, useIntl} from 'react-intl';
+import {Text} from '../../../sharedComponents/Text';
+import {TrackDescriptionField} from './saveTrack/TrackDescriptionField';
+import {NavigationProp} from '@react-navigation/native';
+import {useBottomSheetModal} from '../../../sharedComponents/BottomSheetModal';
+
+const m = defineMessages({
+  newTitle: {
+    id: 'screens.SaveTrack.track',
+    defaultMessage: 'Track',
+    description: 'Category title for new track screen',
+  },
+  detailsButton: {
+    id: 'screens.SaveTrack.TrackEditView.saveTrackDetails',
+    defaultMessage: 'Details',
+    description: 'Button label for check details',
+  },
+  photoButton: {
+    id: 'screens.SaveTrack.TrackEditView.saveTrackCamera',
+    defaultMessage: 'Camera',
+    description: 'Button label for adding photo',
+  },
+});
+
+export const SaveTrackScreen: React.FC<{navigation: NavigationProp<any>}> = ({
+  navigation,
+}) => {
+  const {formatMessage: t} = useIntl();
+  const [description, setDescription] = useState('');
```

Comment:
We want to persist the description in case the user closes their phone. instead of using `setDescription` use                 [updateTags('notes', newVal);](https://github.com/digidem/comapeo-mobile/blob/dd9101b1765e5d3398b0ab6f5a231a0ba7f049f9/src/frontend/screens/ObservationEdit/DescriptionField.tsx#L26) and instead of description use `[const notes = usePersistedDraftObservation(store => store.value?.tags.notes)](https://github.com/digidem/comapeo-mobile/blob/dd9101b1765e5d3398b0ab6f5a231a0ba7f049f9/src/frontend/screens/ObservationEdit/DescriptionField.tsx#L18C3-L18C80)`